openedx/core/djangoapps/content_libraries/tests/test_command_content_libraries_import.py

```diff
@@ -0,0 +1,106 @@
+"""
+Tests for managent command "importcourseware".
```

Comment:
`management`. And the mgmt command is called `content_libraries_import`.

Reply:
+1